src/libraries/System.Drawing.Common/src/System/Drawing/Imaging/EncoderParameter.cs

```diff
@@ -154,7 +154,7 @@ public EncoderParameter(Encoder encoder, int numerator, int denominator)
 
             _parameterValueType = EncoderParameterValueType.ValueTypeRational;
             _numberOfValues = 1;
-            int size = Marshal.SizeOf(typeof(int));
+            int size = sizeof(int);
             _parameterValue = Marshal.AllocHGlobal(2 * size);
```

Comment:
In cases like this, do we even need the local? I can't see what comes after it in the GitHub diff.

Reply:
We do. The EncoderParameter type has `[StructLayout(LayoutKind.Sequential)]` layout and it is used directly with managed interop.

Reply:
I was referring to the `size` local.  Not sure if we're talking about the same thing.  If it is used, we should be able to make `int size = ...;` => `const int Size = ...;`

Reply:
Ah ok. You are right. This local should not be needed.